src/Microsoft.AspNetCore.Authentication.OAuth/OAuthHandler.cs

```diff
@@ -177,7 +172,12 @@ public OAuthHandler(HttpClient backchannel)
                 throw new ArgumentNullException(nameof(context));
             }
 
-            var properties = new AuthenticationProperties(context.Properties);
+            var properties = new AuthenticationProperties(context.Properties)
+            {
+                // Review: should we add an option to configure the state lifetime?
+                ExpiresUtc = Options.SystemClock.UtcNow.AddMinutes(15)
```

Comment:
Yes. It didn't used to expire at all before. `TimeSpan? RoundTripTimeout { get; set; } = TimeSpan.FromMinutes(15);`?